Tests/SwiftDocCTests/DocumentationService/ConvertService/ConvertServiceTests.swift

```diff
@@ -343,7 +340,7 @@ class ConvertServiceTests: XCTestCase {
         }
         
         request.knownDisambiguatedSymbolPathComponents = [
-            "s:5MyKit0A5ClassC10myFunctionyyF": ["MyClass-swift.class", "myFunction()-swift.type.method"],
+            "s:5MyKit0A5ClassC10myFunctionyyF": ["MyClass-swift.class", "myFunction()-swift.method"],
```

Comment:
This disambiguation tries to change the type of this symbol from an instance method to a type method. This isn't valid.